hathor/nanocontracts/resources/blueprint.py

```diff
@@ -90,7 +90,7 @@ def render_GET(self, request: 'Request') -> bytes:
 
         try:
             blueprint_class = self.manager.tx_storage.get_blueprint_class(blueprint_id)
-        except BlueprintDoesNotExist:
+        except (BlueprintDoesNotExist, OCBBlueprintNotConfirmed):
             request.setResponseCode(404)
             error_response = ErrorResponse(success=False, error=f'Blueprint not found: {params.blueprint_id}')
```

Comment:
Should the error message be different for the new exception? Is it not a problem to return "Blueprint not found" in this error?